### DISCUSSION TITLE
Limit concurrent instances, like Scale on v1

### DISCUSSION THREAD

**Discussion by glococo** [vercel/vercel#4047] [General] [answered]
Hi, I have an issue with a WebRTC rest API,

I can't save in memory few signal bytes of webRTC data because the next peer could not connect to the same instance. Thus leaving two clients connected to 2 different instances for sharing few signal bytes.
At this point, I have only one option, use an "external database", which result in a big waste of resources, spent a lot of milliseconds in round-trip to read few vars, and just forward it.

Please, consider enable again the "scale" limit option or anything to limit instances to just one.

Thanks a lot.
ps: I also wrote on a similar issue https://github.com/zeit/now/issues/1639

**Comment by coetry** [comment, marked as answer]
Serverless Functions are a completely different concept than v1 containers, so the same command and concepts like manual scaling can't be carried over. 

The concurrency for functions are handled automatically and if an endpoint is hit at the same time by other users, a new function will serve the second request without any shared state. 

Serverless Functions may not be the best solution for your WebRTC backend server.
